description/description.shacl.ttl

```diff
@@ -361,6 +373,48 @@
 <#PhysicalRepresentationNodeShape> a sh:NodeShape ; 
     sh:targetClass haObj:PhysicalRepresentation ;
     sh:property <#identifierShape>, <#nameShape>, <#descriptionShape>,
+    # Of bij IE?
```

Comment:
Dit is semantisch correct, maar of het in de praktijk niet beter bij de IE zit moet blijken. Laat het voorlopig maar staan